apps/api/src/auth/auth.service.ts

```diff
@@ -190,6 +194,28 @@ export class AuthService {
       Logger.error(response)
       return response
     }
+
+    // Add to marketing platform
+    if (registerDto.newsletter)
+      try {
+        // Add email to general marketing notifications list
+        const mainList = this.config.get('sendgrid.marketingListId')
+        if (mainList)
+          await this.marketingNotificationsService.addContactsToList({
+            contacts: [
+              {
+                email: person.email,
+                first_name: person?.firstName || '',
+                last_name: person?.lastName || '',
+              },
+            ],
+            list_ids: [mainList],
+          })
+      } catch (e) {
+        Logger.error('Failed to subscribe email', e)
+      }
+
+    return person
```

Comment:
this code seems like specific to the NotificationService - let's move it there, because the auth service shouldn't know about list configs